rs/crypto/secp256k1/tests/tests.rs

```diff
@@ -159,9 +192,16 @@ fn should_accept_bip340_signatures_that_we_generate() {
 
         let mut msg = vec![0u8; len];
         rng.fill_bytes(&mut msg);
-        let sig = sk.sign_message_with_bip340(&msg, &mut rng);
 
+        let sig = sk.sign_message_with_bip340(&msg, &mut rng);
         assert!(pk.verify_bip340_signature(&msg, &sig));
+
+        for ttr_len in [0, 32] {
```

Comment:
Hmm, shouldn't this be another test? Something like `should_accept_bip341_signatures_that_we_generate`?

Reply:
Eh, they have very much the same structure but I guess it does make sense to split it out to a distinct test case.